### PR TITLE
III-3706

### PR DESCRIPTION
**Pull request by jdrieghe**
### Removed
- Dependency on `phing/phing` for the CI pipeline
 
---
Ticket: https://jira.uitdatabank.be/browse/III-3706

### CODE REVIEW

File: src/CultureFeed/CultureFeedApiKeyAuthenticator.php [hunk 1]
```diff
@@ -27,27 +27,24 @@ class CultureFeedApiKeyAuthenticator implements ApiKeyAuthenticatorInterface
     private $includePermissions;
 
     /**
-     * @param \ICultureFeed $cultureFeed
-     * @param ConsumerWriteRepositoryInterface $consumerWriteRepository
      *   CultureFeed returns a consumer detail while authenticating.
      *   This consumer will be written to the injected write repository as a
      *   form of caching.
```
Comment (bertramakers) [inline src/CultureFeed/CultureFeedApiKeyAuthenticator.php]: Nit: the description that's now left was for the removed `@param`. If we leave it in, we should probably change the indentation?

**Review by bertramakers — APPROVED** [review]
👍

File: src/CultureFeed/CultureFeedApiKeyAuthenticator.php (hunk 1 above)
Comment (jdrieghe) [reply src/CultureFeed/CultureFeedApiKeyAuthenticator.php]: This is part of a merge from what's already in `main`, so probably better to fix it there immediately.

Comment (bertramakers) [reply src/CultureFeed/CultureFeedApiKeyAuthenticator.php]: Yeah nevermind it's super minor and not worth it then 😄

**Pull request merged**